src/v3/preact.config.js

```diff
@@ -52,16 +53,14 @@ export default {
       (plugin) => !(plugin instanceof MiniCssExtractPlugin),
     );
 
-    config.plugins.push(
-      new DefinePlugin({
-        // for OktaSignIn.__version, AuthContainer[data-version]
-        VERSION: JSON.stringify(gitRevisionPlugin.version()),
-        // for OktaSignIn.__commit, AuthContainer[data-commit]
-        COMMITHASH: JSON.stringify(gitRevisionPlugin.commithash()),
-        // for v2/util/Logger file
-        DEBUG: env !== 'production',
-      }),
-    );
+    config.plugins.push(new DefinePlugin({
+      // for OktaSignIn.__version, AuthContainer[data-version]
+      VERSION: JSON.stringify(version),
```

Comment:
`git-revision-webpack-plugin` uses `git describe`

From the [README](https://github.com/pirelenito/git-revision-webpack-plugin#usage)

> It outputs a VERSION based on [git-describe](http://www.git-scm.com/docs/git-describe) such as:
> v0.0.0-34-g7c16d8b

The value set using this plugin was `okta-signin-widget-6.5.0-54-gcf0c92539` instead of `okta-signin-widget-0.0.0-...` wherever the `DefinePlugin` replaced the string.